app/JACK_Audio_Connection_Kit_sound_server/doc/README.head.md

```diff
@@ -0,0 +1,11 @@
+Monitoring of the [JACK Audio Connection Kit sound server](http://www.jackaudio.org).
+
+## Usage
```

Comment:
Consider moving this to the docs template. 3. could be part of a `## Configuration` section above usage if we want to keep everything specific in `README.head.md`. We could also add a `README.usage.md` that overides the templated usage instructions for these rare cases.

Reply:
Removed, added to the `README.Usage.md` override and updated with the help of the new usage generation (#74).